src/util/OptionsRegistry.v3

```diff
@@ -89,11 +89,11 @@ class OptionGroup(name: string, o: BasicOptions) {
 	def parseEmpty(str: string) -> bool {
 		return str == null;
 	}
-	def print(buf: StringBuilder) {
+	def print(buf: StringBuilder, indent: string) {
```

Comment:
Since this already indents a little, maybe take the indent as an int here?

Reply:
I ended up making this a flag `is_monitor_opt` and generates the monitor options help text exactly as expected